library/cpp/.clang-format

```diff
@@ -0,0 +1,8 @@
+---
+BasedOnStyle: Chromium
+IndentWidth: 4
+AllowShortIfStatementsOnASingleLine: AllIfsAndElse
+ColumnLimit: 0
+InsertNewlineAtEOF: true
+MaxEmptyLinesToKeep: 2
+---
```

Comment:
Default .clang-format for tests. Any style suggestions should ideally come here.